plugin/pkg/kubectl/utils/ui.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import "fmt"
+
+// Green will print the specified string in green text
+func Green(str string) string {
+	return fmt.Sprintf("\x1b[32;1m%s\x1b[0m", str)
```

Comment:
Doesn't work on Windows.

Reply:
There are several packages for printing colorized output. For example: https://godoc.org/github.com/fatih/color

Can you use one of those instead of rolling your own here